### PR TITLE
add dash script; add dash to player prefab

### PR DESCRIPTION
**Pull request by ameijboom**

### CODE REVIEW

File: Assets/Scripts/Player/Dash.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MonoBehaviour
+{
+    [SerializeField] private float _dashPower;
+    private Rigidbody _rb;
+    private Quaternion _direction;
```
Comment (TechnicJelle) [inline Assets/Scripts/Player/Dash.cs:9]: Why are you storing this?
Why retrieve this every single frame, when you can also just retrieve it only when the player actually dashes?

Comment (TechnicJelle) [inline Assets/Scripts/Player/Dash.cs]: Why are you storing the direction?
Why retrieve this every single frame, when you can also just retrieve it only when the player actually dashes?

**Review by TechnicJelle — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/Dash.cs (hunk 1 above)
Comment (ameijboom) [reply Assets/Scripts/Player/Dash.cs]: Good point, will change it

**Review by TechnicJelle — COMMENTED** [review]
lgtm

**Pull request merged**